git_machete/client.py

```diff
@@ -979,17 +979,17 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
                             # the above call to 'fork_point_hash'.
                             fp_branches_formatted: str = " and ".join(
                                 sorted(underline(lb_or_rb) for lb, lb_or_rb in fork_point_branches_cached[branch]))
-                            fp_suffix: str = " %s %s %s seems to be a part of the unique history of %s" % \
-                                             (colored(utils.get_right_arrow(), AnsiEscapeCodes.RED),
-                                              colored("fork point ???", AnsiEscapeCodes.RED),
-                                              "this commit" if opt_list_commits_with_hashes else f"commit {commit.short_hash}",
-                                              fp_branches_formatted)
+                            right_arrow = colored(utils.get_right_arrow(), AnsiEscapeCodes.RED)
+                            fork_point = colored("fork point ???", AnsiEscapeCodes.RED)
+                            fp_suffix: str = f' {right_arrow} {fork_point}\
```

Comment:
Hmmm won't there be a space missing between `???` and `this commit`? which would be a change from the original impl 🤔